.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,84 @@
+name: Bug Report
+description: Help us improve.
+labels: ["bug"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+
+        Arelle is distributed in a few different ways and can be run in a variety of environments.
+        Providing details on how you're using Arelle will significantly decrease the amount of time to figure out what's wrong.
+
+        When relevant please try to provide a document that reproduces the issue.
+  - type: textarea
+    id: what-happened
+    attributes:
+      label: What happened?
+      description: And what did you expect to happen?
+    validations:
+      required: true
+  - type: textarea
+    id: documents
+    attributes:
+      label: Documents
+      description: If possible, please attach any documents that cause the bug.
+      placeholder: Drag and drop documents here.
+  - type: input
+    id: cli-command
+    attributes:
+      label: If running from the command line, what command did you run?
+      placeholder: e.g. `python -m arelleCmdLine --plugins validate/EFM --validate --file xbrl.zip`
+  - type: dropdown
```

Comment:
This is neat, can you make the template conditional based on the response given here? If so I wouldn't ask the previous question unless they select CLI. Also might want a checkbox instead, I am not sure these are always discreet